tests/acceptance/sign-up-test.js

```diff
@@ -0,0 +1,31 @@
+import { module, test } from 'qunit';
+import { click, fillIn, visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import Service from '@ember/service';
+
+class StoreStub extends Service {
+  createRecord(modelName, obj) {
+    if (modelName === 'user' && obj.email) {
+      return { email: obj.email, isValid: true, save: () => true }
+    }
+  }
+}
```

Comment:
Since this stub also stubs the `save` function so it returns true, it prevents the test from actually doing a network request in the tests, which we for sure don't want to do.